build/config/dcheck_always_on.gni

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+declare_args() {
+  # Enables DCHECKs to be built-in, but to default to being non-fatal/log-only.
+  # DCHECKS can then be set as fatal/non-fatal via the DCheckIsFatal feature.
+  # See crbug.com/596231 for details on how this is used.
+  dcheck_is_configurable = false
```

Comment:
This does nothing in our buildroot. Let's always set both to false.

Reply:
Updated to make this always false. I'm leaving `dcheck_always_on` in a declare_args since it was already an argument before the patch, and I'd rather not have potential fallout from removing an argument as part of this patch.